index.js

```diff
@@ -290,7 +290,7 @@ exports.register = function (baseModule, options) {
 
 	// find the nearest package.json
 
-	var pkg = exports.findPackage(baseModule, true);
+	var pkg = exports.findPackage(baseModule, options.strictCheck !== false);
```

Comment:
Could you add a mention of this option to both the JSDoc above, and the ReadMe?

Reply:
done